src/server/system_services/master_key_manager.js

```diff
@@ -10,7 +10,8 @@ const dbg = require('../../util/debug_module')(__filename);
 const SensitiveString = require('../../util/sensitive_string');
 const LRUCache = require('../../util/lru_cache');
 
-const ROOT_KEY = 'noobaa-root-master-key';
+// object id of root key
+const ROOT_KEY = '000000000000000000000000';
```

Comment:
I've used dummies for triggers:
```
dispatch_obj.obj_id = '10101010aaaabbbbccccdddd';
```
Maybe assigning a total 0 a bit riskier, we can re-use similar logic with the change of 0 and 1.
```            
const ROOT_KEY =  '00000000aaaabbbbccccdddd';
```
Or any other suggestion by you obviously.

Reply:
fixed